operations/cancellationpolicies.md

```diff
@@ -91,22 +94,21 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `RateGroupId` | string | required | Unique identifier of the rate group the cancellation policy belongs to. |
 | `CreatedUtc` | string | required | Date and time of the cancellation policy creation in UTC timezone in ISO 8601 format. |
 | `UpdatedUtc` | string | required | Date and time of the cancellation policy update in UTC timezone in ISO 8601 format. |
-| `Applicability` | string [Applicability](#cancellation-policy-applicability) | required | Applicability mode of the cancellation policy. |
-| `FeeExtent` | array of string [Fee Extent](#fee-extent) | required | Extent for the cancellation fee, i.e. what should be in scope for the automatic payment. |
-| `ApplicabilityOffset` | string | required | Offset for order start (assuming Applicability is set to `Start`) from which the fee is applied. |
-| `FeeMaximumTimeUnits` | int | required | Maximum number of time units the cancellation fee is applicable to. |
-| `AbsoluteFee` | [Currency value](../operations/accountingitems.md#currency-value) | optional | Absolute value of the fee. |
-| `RelativeFee` | decimal | required | Relative value of the fee, as a percentage of the reservation price. |
+| `Applicability` | [Cancellation Policy Applicability](cancellationpolicies.md#cancellation-policy-applicability) | required | Applicability mode of the cancellation policy. |
+| `FeeExtent` | array of [Cancellation Fee Extent](cancellationpolicies.md#cancellation-fee-extent) | required | Extent for the cancellation fee, i.e. what should be in scope for the automatic payment. |
+| `ApplicabilityOffset` | string | required | Offset for order start (assuming Applicability is set to Start) from which the fee is applied. |
+| `FeeMaximumTimeUnits` | integer | required | Maximum number of time units the cancellation fee is applicable to. |
+| `AbsoluteFee` | [Currency value (ver 2023-02-02)](_objects.md#currency-value-ver-2023-02-02) | required | Absolute value of the fee. |
+| `RelativeFee` | number | required | Relative value of the fee, as a percentage of the reservation price. |
+| `IsActive` | boolean | required | Whether the cancellation policy is still active. |
 
 #### Cancellation Policy Applicability
 
-* `Creation` - Cancellation fee is applicable from the time of creating the reservation.
-* `Start` - Cancellation fee is applicable as soon as the reservation starts, i.e. at arrival time.
-* `StartDate` - Cancellation fee is applicable on the date the reservation starts, i.e., at midnight.
-* ...
```

Comment:
We need to be careful about these. 
Historically, enums ending with ... were defined as an extensible enums implying that we could add more values there, whereas with a removal of ... we are saying it's going to be a locked enum with no possibility to extend. Is this intended?

Reply:
I believe that @jnv mentioned that the three dots annotation of extensible enums wont be used in the generated documentation

Reply:
@jnv Don't we want to consider adding tooling to automated generator, so that we can potentially add this '...' concept to the docs, so we don't lose an option to extend enums in future?

Reply:
@vitezp I feel like answering this for 6th time. We've discussed this with Mike and intentionally decided to drop the ellipsis. We want to reintroduce it but in different manner, at OAS level so codegens understand it and provide a fallback value (which'd become a part of the contract as well).

Reply:
Here you go @vitezp: https://mews.atlassian.net/wiki/spaces/CON/pages/151683883/Generating+reference+API+documentation#The-generated-documentation-doesn%E2%80%99t-contain-ellipsis-(%E2%80%A6)-for-%E2%80%9Cextensible-enums%E2%80%9D%2C-is-this-correct%3F 😉